uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1007.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1007 {
+
+  /**
+   * Make the Line Manager/Line Manager is Relationship
```

Comment:
Very minor, but just wondering why you added a line break here, when line 16 below is longer